pkg/installer/install

```diff
@@ -64,8 +64,8 @@ logmsg() {
 }
 
 pause() {
-   echo "Pausing before $1. Entering shell. Type 'exit' to proceed to $1"
-   sh
+   echo "Pausing before $1. Entering shell. Type 'exit' to proceed to $1" > /dev/console 2>&1
+   sh < /dev/console > /dev/console 2>&1
```

Comment:
We are 100% sure that `/dev/console` is available and has the proper permissions, right?

Reply:
I think so, /dev/console always points to the system's console, also, if you look a few lines above (https://github.com/lf-edge/eve/pull/4299/files/30b4e8972e3ee04d9114c6c9108d69b9ebd1fc21#diff-d2570dd09cc66cad9c33cbf655c47811e5959ad5d2df67fabfb2ca2d44b293f7R63) that's how the installer script is printing information to the terminal and the log file.....

Reply:
Ah, yeah, makes sense